test/integration/full/document-title/frames/level2-a.html

```diff
@@ -8,4 +8,4 @@
 <body>
 	Hi
 </body>
-</html>
+</html>
```

Comment:
Same as above and below–can you revert these integration test files to re-introduce the line ending